lib/fizzy/execute.cpp

```diff
@@ -19,6 +20,11 @@ namespace
 // code_offset + imm_offset + stack_height
 constexpr auto BranchImmediateSize = 3 * sizeof(uint32_t);
 
+constexpr auto ABS_MASK = 0x7fffffff;
```

Comment:
Please, use the following naming convention. Also specify the type explicitly (the rules what type literals have are confusing).
```suggestion
constexpr uint32_t F32AbsMask = 0x7fffffff;
```